### PR TITLE
Added ordinal ranking

### PR DESCRIPTION
**Pull request by TwelveNights**

### CODE REVIEW

File: imports/ui/Body.js [hunk 1]
```diff
@@ -15,26 +14,31 @@ import './Index.js';
 import './HongKongNewGame.js';
 import './JapaneseNewGame.js';
 
-Template.JapaneseRanking.helpers({
-	japanesePlayers() {
-		return Players.find({}, {sort: { japaneseElo: -1}});
-	},
+var index;
+
+function initPlayer(key) {
```
Comment (Futochu) [inline imports/ui/Body.js]: Would it be better to specify what sort of sorting we want if we want to sort by other metrics layer?

File: imports/ui/Body.js [hunk 2]
```diff
@@ -15,26 +14,31 @@ import './Index.js';
 import './HongKongNewGame.js';
 import './JapaneseNewGame.js';
 
-Template.JapaneseRanking.helpers({
-	japanesePlayers() {
-		return Players.find({}, {sort: { japaneseElo: -1}});
-	},
+var index;
```
Comment (Futochu) [inline imports/ui/Body.js]: Try to avoid global variables, especially one with such a generic name

File: imports/ui/Body.js [hunk 3]
```diff
@@ -15,26 +14,31 @@ import './Index.js';
...
+
+function initPlayer(key) {
+	index = 1;
+
+	var sort = {};
+	sort[key] = -1;
```
Comment (Futochu) [inline imports/ui/Body.js]: We should be able to specify other sort directions if, say, we want to sort by chombo# ascending.

File: imports/ui/HongKongRanking.html [hunk 4]
```diff
@@ -2,16 +2,16 @@
 	<h2>Hong Kong Mahjong ELO Rankings</h2>
 	<table>
 		<tr>
-			<th align=center width=200px>Players</th><th width=50px></th><th>ELO</th>
+			<th align=center width=50px>Rank</th>
```
Comment (Futochu) [inline imports/ui/HongKongRanking.html]: Ignore the fact you just copied me, I think we should commit this and wait for frontend to fix

File: imports/ui/Body.js [hunk 5]
```diff
@@ -1,5 +1,4 @@
 import { Template } from 'meteor/templating';
-
```
Comment (Futochu) [inline imports/ui/Body.js]: any reason?

File: imports/ui/Body.js [hunk 6]
```diff
@@ -15,26 +14,31 @@ import './Index.js';
...
-	},
+var index;
+
+function initPlayer(key) {
+	index = 1;
+
+	var sort = {};
+	sort[key] = -1;
+	return Players.find({}, sort);
+}
```
Comment (Futochu) [inline imports/ui/Body.js]: Please end functions with a semicolon

File: server/main.js [hunk 7]
```diff
@@ -33,10 +33,9 @@ Meteor.startup(() => {
   			japaneseWinPointsTotal: 0, //added
   			japaneseWinDoraTotal: 0, //added
   			japaneseRiichiTotal: 0, //added
-			japaneseWinRiichiTotal: 0, //added
+  			japaneseWinRiichiTotal: 0, //added
   			japaneseChomboTotal: 0, //added
   			japaneseBankruptTotal: 0, //added
-
```
Comment (Futochu) [inline server/main.js]: I'm fine with the indentation fixes, but why this?

**Comment by Futochu** [comment]
there are also inline comments I posted before I realised reviews were a thing

File: imports/ui/Body.js (hunk 5 above)
Comment (TwelveNights) [inline imports/ui/Body.js]: Nah, just combining the '{...}' imports

**Comment by TwelveNights** [comment]
Move to #6

**Pull request closed without merging**